rocm-validation-suite/PKGBUILD

```diff
@@ -10,8 +10,15 @@ license=('MIT')
 depends=('pciutils' 'doxygen' 'rocblas' 'rocm-smi-lib64' 'git')
 makedepends=('cmake')
 options=(!staticlibs strip)
-source=("$pkgname-$pkgver.tar.gz::https://github.com/ROCm-Developer-Tools/ROCmValidationSuite/archive/rocm-$pkgver.tar.gz")
-sha256sums=('17662028a4485b97e3ccaad5e94d20aaa2c3e9e3f741c7ebbf0f8b4cdebcc555')
+source=("$pkgname-$pkgver.tar.gz::https://github.com/ROCm-Developer-Tools/ROCmValidationSuite/archive/rocm-$pkgver.tar.gz"
+        "action.patch")
+sha256sums=('17662028a4485b97e3ccaad5e94d20aaa2c3e9e3f741c7ebbf0f8b4cdebcc555'
+            '16059a24ae9c01a7e77e1323a2437b9222998d3ac19d7c3367bcf821da60ecd9')
+
+prepare() {
+ cd $srcdir
+ patch ROCmValidationSuite-rocm-$pkgver/rcqt.so/src/action.cpp action.patch 
```

Comment:
It seems like you are still not following the convention.

Maybe taking a look at the ArchWiki for guidance: https://wiki.archlinux.org/index.php/Patching_packages#Creating_patches

The key thing is not patching the file but the directory.

Oh also indentation.